ocs/ocp.py

```diff
@@ -204,3 +205,50 @@ def wait_for_resource(
                 return True
 
         return False
+
+    def exec_cmd_on_pod(self, pod_name, command):
+        """
+        Execute a command on a pod (e.g. oc rsh)
+
+        Args:
+            pod_name (str): The pod on which the command should be executed
+            command (str): The command to execute on the given pod
+
+        Returns:
+            Munch Obj: This object represents a returned yaml file
+        """
+        rsh_cmd = f"rsh {pod_name} "
+        rsh_cmd += command
+        return self.exec_oc_cmd(rsh_cmd)
+
+
+# the following functions location is temporary, will be changed to ocs class
+def get_ceph_tools_pod():
+    """
+    Get the Ceph tools pod
+
+    Returns:
+        str: The Ceph tools pod name
+    """
+    ocp_pod_obj = OCP(kind='pods', namespace=defaults.ROOK_CLUSTER_NAMESPACE)
+    ct_pod = ocp_pod_obj.get(
+        selector='app=rook-ceph-tools'
+    ).toDict()['items'][0]['metadata']['name']
```

Comment:
Look as Coady accessing this directly via jsonpath [here](https://github.com/red-hat-storage/ocs-ci/blob/master/tests/ecosystem/deployment/test_ocs_basic_install.py), we can then reuse this function in installation as well.

Reply:
Thanks @petr-balogh. 
You refer to:
```python
    tools_pod = run_cmd(
        f"oc -n {namespace} get pod -l 'app=rook-ceph-tools' "
        f"-o jsonpath='{{.items[0].metadata.name}}'"
    )
```

Is there any advantage of using it over what I did?

Reply:
just that the you will safe toDict call and you will get directly the name from oc command. And if no such pod found that you will get raised exception from run_cmd.  But I don't mind, anyway, it's merged already :) :+1: